control.py

```diff
@@ -0,0 +1,237 @@
+from examples.declarations.export_prepro import csv_and_json
+
+
+control_structure = {
+    "app_configuration": {
+        "state_format": "vmi13",  # options: fdm, vmi12, vmi13, xml, gpkg
+        # "state_input_container": "csv",  # Only relevant with fdm state_format. Options: pickle, json
+        # "state_output_container": "csv",  # options: pickle, json, csv, null
+        # "derived_data_output_container": "pickle",  # options: pickle, json, null
+        "formation_strategy": "partial",
+        "evaluation_strategy": "depth",
+        "run_modes": ["preprocess", "export_prepro", "simulate", "postprocess", "export"]
+    },
+    "preprocessing_operations": [
+        "scale_area_weight",
+        "generate_reference_trees",  # reference trees from strata, replaces existing reference trees
+        "filter"
+        # "supplement_missing_tree_heights",
+        # "supplement_missing_tree_ages",
+        # "generate_sapling_trees_from_sapling_strata"
+    ],
+    "preprocessing_params": {
+        "generate_reference_trees": [
+            {
+                "n_trees": 10,
+                "method": "weibull",
+                "debug": False
+            }
+        ],
+        "filter": [
+            {
+                "remove trees": "sapling or stems_per_ha == 0",
+                "remove stands": "site_type_category == 0", # not reference_trees
+                "remove stands": "site_type_category == None"
```

Comment:
Since the config is in Python now, it would make sense to make these real functions, e.g. `"remove trees": lambda t: t.sapling or t.stems_per_ha == 0`. Same with the operations, e.g. `from metsi.something import first_thinning, even_thinning`. Then we can also eliminate all the `operation_params` ceremony because you can just pass them as normal function params. (it's probably best to leave this change for a later PR, just pointing it out because it came to my mind while reading this.)

Reply:
You are absolutely right. We should change this in the future. This PR is too huge already 🤣